core/controllers/classifier.py

```diff
@@ -99,6 +99,7 @@ def post(self):
         signature = self.payload.get('signature')
         message = self.payload.get('message')
         vm_id = self.payload.get('vm_id')
+
```

Comment:
Please remove if this isn't an intentional part of the PR (make sure to check the diff on GitHub).

Reply:
OK.

Reply:
Done